src/lib/Form/DataTransformer/ContentTransformer.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace EzSystems\EzPlatformAdminUi\Form\DataTransformer;
+
+use eZ\Publish\API\Repository\ContentService;
+use eZ\Publish\API\Repository\Exceptions\NotFoundException;
+use eZ\Publish\API\Repository\Values\Content\Content;
+use Symfony\Component\Form\DataTransformerInterface;
+use Symfony\Component\Form\Exception\TransformationFailedException;
+
+class ContentTransformer implements DataTransformerInterface
+{
+    /** @var \eZ\Publish\API\Repository\ContentService */
+    protected $contentService;
+
+    /**
+     * @param \eZ\Publish\API\Repository\ContentService $contentService
+     */
+    public function __construct(ContentService $contentService)
+    {
+        $this->contentService = $contentService;
+    }
+
+    /**
+     * Transforms a domain specific ContentInfo object into a Content's ID.
+     *
+     * @param \eZ\Publish\API\Repository\Values\Content\Content|null $value
+     *
+     * @return int|null
+     */
+    public function transform($value)
+    {
+        if (null === $value) {
+            return null;
+        }
+
+        if (!$value instanceof Content) {
+            throw new TransformationFailedException('Expected a ' . Content::class . ' object.');
+        }
+
+        return $value->id;
+    }
+
+    /**
+     * Transforms a Content's ID integer into a domain specific Content object.
+     *
+     * @param string|null $value
+     *
+     * @return \eZ\Publish\API\Repository\Values\Content\Content|null
+     *
+     * @throws \eZ\Publish\API\Repository\Exceptions\UnauthorizedException
+     */
+    public function reverseTransform($value): ?Content
+    {
+        if (empty($value)) {
+            return null;
+        }
+
+        if (!is_numeric($value)) {
```

Comment:
It's not a big deal, but better choice for `int` validation will be (filter_var)[http://php.net/manual/en/function.filter-var.php] or (ctype_digit)[http://php.net/manual/en/function.ctype-digit.php]  `is_numeric` also returns `true` for '3e4', '12.6' etc.